### PR TITLE
Workflow Automated Test - LRQA-71621 | master

### PR DESCRIPTION
**Pull request by Italo-gabriel**
https://issues.liferay.com/browse/LRQA-71621

### CODE REVIEW

**Comment by Italo-gabriel** [comment]
ci:test:sf

**Comment by Italo-gabriel** [comment]
ci:test:sf

**Comment by alanafergs** [comment]
Just started reviewing :)

File: modules/apps/calendar/calendar-web-test/src/testFunctional/tests/CalendarConfiguration.testcase [hunk 1]
```diff
@@ -60,9 +60,64 @@ definition {
...
+        CalendarEvent.viewTimesViaDetails(
+            endDateTime = "03:15 PM",
+            startDateTime = "03:00 PM");
+
+        Navigator.gotoPage(pageName = "Calendar Page");
 
-		// TODO LRQA-71621 CanConfigureEventsDefaultDuration pending implementation
+        CalendarNavigator.gotoConfiguration();
 
+        CalendarConfiguration.viewSelectDefaultDuration(defaultDuration = "15 Minutes");
```
Comment (lucasperj) [inline modules/apps/calendar/calendar-web-test/src/testFunctional/tests/CalendarConfiguration.testcase]: In this test, you are manually entering the desired time and the time space that we are trying to verify(15, 45, 60 minutes). You can look for a way to insert the event without directly interfering with the time, being able to just check if the default time was modified during the test. 
You can take a look at the macro 
```
CalendarScheduler.addLocalizedEvent(
			eventTitle = "Calendar Event Title",
			saveButton = "Save",
			viewType = "Week");
```

**Review by lucasperj — CHANGES_REQUESTED** [review]

**Pull request closed without merging**